MkDocs/monitor_csv_importer_8.6/docs/prereqs.md

```diff
@@ -30,7 +30,7 @@ Monitor API credentials:
 Monitor DB2 Warehouse credentials:
 
 1.  Hostname
-2.  Database, usually BLUDB
+2.  Database, usually `BLUDB`
```

Comment:
Explain how in MAS it also requires downloading and installing Database.pem file.

Reply:
Not in this case.